src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -24,6 +26,29 @@ public class PreferencesViewModel : ViewModelBase
         private bool nodeAutocomplete;
         private bool enableTSpline;
 
+        private PreferenceSettings _preferenceSettings;
+        public PreferenceSettings PreferenceSettings
+        {
+            get { return _preferenceSettings; }
+            set
+            {
+                _preferenceSettings = value;
+                RaisePropertyChanged(nameof(PreferenceSettings));
+            }
+        }
+
+        private DynamoPythonScriptEditorTextOptions editTextOptions;
+        public DynamoPythonScriptEditorTextOptions PythonScriptEditorTextOptions
+        {
+            get { return editTextOptions; }
+            set
+            {
+                editTextOptions = value;
```

Comment:
Do we need to raise property change here?

Reply:
I believe not in this case, as this is only used to access other properties that the user can change on the panel, and this properties will rise the event. (Example in line 247)

Reply:
does it need to be public then?

Reply:
good catch, going over that piece of code again I notice it can be simplified, I will update on the next commit

Reply:
Just updated this, I refactored a little bit that property and one more as it didnt make much sense having a private property to access another private one. Let me know what you think, thanks.